jolie/src/main/java/jolie/OOITBuilder.java

```diff
@@ -500,7 +500,11 @@ public void visit( InputPortInfo n ) {
 				return;
 			}
 			if( protocolExpr instanceof Value || protocolExpr instanceof InlineTreeExpression ) {
-				protocolStr = protocolExpr.evaluate().strValue();
+				try {
+					protocolStr = protocolExpr.evaluate().strValue();
+				} catch( FaultException e ) {
+					throw new AssertionError( "Expression.evaluate() should never throw an exception here" );
```

Comment:
When defining the protocol for example `protocol: http { format = "json" }`. Having function calls inside this definition is not support eg.  `protocol: http { format = trim@StringUtils(" json")}`, (Another error will occur). Because of this I assumed that it is not supposed to be possible that the `protocolExpr `could be a `SolicitResponseExpression`, and thus the FaultException should never happen.